app/lib/reducers/ui.js

```diff
@@ -111,7 +111,7 @@ const reducer = (state = initial, action) => {
         }
 
         if (null != config.colors) {
-          if (state.colors.toString() !== config.colors.toString()) {
+          if (JSON.stringify(state.colors) !== JSON.stringify(config.colors) {
```

Comment:
@mike-engel I think there's a typo here, a missing parenthesis. Should be 
```javascript
if (JSON.stringify(state.colors) !== JSON.stringify(config.colors)) {
```
instead.